packages/babel-plugin/src/styled/index.tsx

```diff
@@ -124,12 +124,12 @@ export const visitStyledPath = (
   const cssOutput = buildCss(styledData.cssNode, meta);
 
   path.replaceWith(
-    buildStyledComponent({
-      ...meta.state.opts,
-      cssOutput,
-      tag: styledData.tag,
-      parentPath: path as NodePath,
-      scope: path.scope,
```

Comment:
Are we not passing updated `parentPath` and `scope` now 🤔?

Reply:
we are just through metadata instead

Reply:
hmmm I just thought we are passing updated `parentPath` and `scope` separately. No worries, if we are now passing these through meta. This new api looks more clean 🎉